### PR TITLE
more small fixes

### PR DESCRIPTION
**Pull request by jfmherokiller**

### CODE REVIEW

File: include/SKSE/Impl/WinAPI.h [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-	inline const auto     INVALID_HANDLE_VALUE{ reinterpret_cast<void*>(static_cast<std::intptr_t>(-1)) };
-	inline constexpr auto MAX_PATH{ static_cast<std::uint32_t>(260) };
-	inline constexpr auto MEM_RELEASE{ static_cast<std::uint32_t>(0x00008000) };
-	inline constexpr auto PAGE_EXECUTE_READWRITE{ static_cast<std::uint32_t>(0x40) };
+	inline static constexpr auto IMAGE_SCN_MEM_EXECUTE{ static_cast<std::uint32_t>(0x20000000) };
+	inline static constexpr auto IMAGE_SCN_MEM_WRITE{ static_cast<std::uint32_t>(0x80000000) };
+	inline static const auto     INVALID_HANDLE_VALUE{ reinterpret_cast<void*>(static_cast<std::intptr_t>(-1)) };
+	inline static constexpr auto MAX_PATH{ static_cast<std::uint32_t>(260) };
+	inline static constexpr auto MEM_RELEASE{ static_cast<std::uint32_t>(0x00008000) };
+	inline static constexpr auto PAGE_EXECUTE_READWRITE{ static_cast<std::uint32_t>(0x40) };
```
Comment (Ryan-rsm-McKenzie) [inline include/SKSE/Impl/WinAPI.h:10]: This keyword is redundant at namespace scope.

File: include/SKSE/Impl/PCH.h [hunk 2]
```diff
@@ -499,7 +499,7 @@ namespace SKSE
 					fileview = fileview.substr(pos + off);
 				}
 
-				return fmt::format(FMT_STRING("{}({}): {}"), fileview, a_loc.line(), a_msg);
+				return fmt::format("{0}({1}): {2}", fileview, a_loc.line(), a_msg);
```
Comment (Ryan-rsm-McKenzie) [inline include/SKSE/Impl/PCH.h:502]: Why would you remove compile time format checking and make the format more brittle to changes?

File: include/SKSE/Impl/PCH.h [hunk 3]
```diff
@@ -474,7 +474,7 @@ namespace SKSE
 
 		[[noreturn]] inline void report_and_fail(std::string_view a_msg, std::source_location a_loc = std::source_location::current())
 		{
-			const auto body = [&]() {
+			const auto body = [&]() -> std::string {
```
Comment (Ryan-rsm-McKenzie) [inline include/SKSE/Impl/PCH.h:477]: This is unecessary.

Comment (jfmherokiller) [reply include/SKSE/Impl/PCH.h:477]: this one fixes an IDE error wherein the it thinks the function returns void

Comment (Ryan-rsm-McKenzie) [reply include/SKSE/Impl/PCH.h:477]: that's an issue with your ide, the compiler deduces the return type just fine.

**Pull request closed without merging**